psydac/feec/multipatch/tests/tests_feec_multipatch_2d.py

```diff
@@ -0,0 +1,247 @@
+from mpi4py import MPI
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from scipy.sparse.linalg import spsolve
+
+from sympde.topology import Derham
+from sympde.topology import element_of, elements_of
+from sympde.topology import Square
+from sympde.topology import IdentityMapping, PolarMapping
+
+from sympde.expr.expr import LinearForm, BilinearForm
+from sympde.expr.expr import integral
+from sympde.expr      import Norm
+
+from psydac.linalg.iterative_solvers import cg, pcg
+from psydac.linalg.utilities         import array_to_stencil
+from psydac.fem.basic                import FemField
+
+from psydac.feec.multipatch.fem_linear_operators import IdLinearOperator, ComposedLinearOperator
+from psydac.feec.multipatch.api                  import discretize
+from psydac.feec.multipatch.operators            import BrokenMass
+from psydac.feec.multipatch.operators            import ConformingProjection_V0
+from psydac.feec.multipatch.operators            import get_patch_index_from_face
+
+from psydac.api.essential_bc import apply_essential_bc_stencil
+
+comm = MPI.COMM_WORLD
+#==============================================================================
+def run_poisson_2d(solution, f, domain, ncells, degree, tol=1e-9, use_scipy=False):
```

Comment:
this test does not use the new  Feec-multipatch interface, I'm not sure it is very relevant....

Reply:
(same remark for the other tests below)

Reply:
ah yes, those tests are old, I'm going to delete them

Reply:
but it's good to have tests -- why not just updating them with the new feec multipatch interface ?

Reply:
yes, I have added new ones

Reply:
you can see them here [d245568](https://github.com/pyccel/psydac/pull/185/commits/d245568d574c10a34975106be754cc8baf597318)

Reply:
👍